protocol/avdl/keybase1/teams.avdl

```diff
@@ -230,6 +230,10 @@ protocol teams {
 
     // All invitations that are currently active.
     map<TeamInviteID,TeamInvite> activeInvites;
+
+    // Whether this is an open team.
+    boolean open;
+    TeamRole openTeamJoinAs;
```

Comment:
Because of this addition you will want to bust the team cache by incrementing the `diskStorageVersion` in `teams/storage.go`. This is so that a future client running code before this commit who caches teams without having processed settings will, on upgrade, reload sigchains with this new code.